src/snapshot/Cargo.toml

```diff
@@ -8,7 +8,7 @@ autobenches = false
 serde = ">=1.0.27"
 serde_derive = ">=1.0.27"
 bincode = "1.2.1"
-versionize = { git = "https://github.com/firecracker-microvm/versionize", tag = "v0.1.1" }
+versionize = { git = "https://github.com/firecracker-microvm/versionize", tag = "v0.1.0" }
```

Comment:
Is this intentional?

Reply:
You can leave it like that, we will later propagate that upgrade to all deps that need it.

Reply:
I had a compile problem when the `snapshot` crate was using `versionize 0.1.1` and the other crates `versionize 0.1.0` and I saw that there is another PR which bumps the version, so I took this shortcut 😅